chromium_src/chrome/browser/component_updater/component_updater_utils.cc

```diff
@@ -0,0 +1,22 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "../../../../../chrome/browser/component_updater/component_updater_utils.cc"
+
+#include "base/callback.h"
+#include "chrome/browser/browser_process.h"
+#include "components/component_updater/component_updater_service.h"
+
+namespace component_updater {
+
+void BraveOnDemandUpdate(const std::string& component_id) {
```

Comment:
we already have `BraveOnDemandUpdater` for this

Reply:
Are you saying to do this as a static function in `BraveOnDemandUpdater`?

Reply:
I'm saying that we should be using BraveOnDemandUpdater instead of this new method for anything that was previously using `ComponentsUI::OnDemandUpdate`. You don't need a new static function, it already does this